rpc/api_contract_test.go

```diff
@@ -188,33 +138,33 @@ func TestStorageAt(t *testing.T) {
 	testConfig := beforeEach(t)
 
 	type testSetType struct {
-		ContractHash   string
+		ContractHash   Address
 		StorageKey     string
-		BlockHashOrTag string
+		BlockHashOrTag BlockIDOption
```

Comment:
```suggestion
		BlockHashOrTag BlockHashOrTag
```

I might rename this to match the spec

Reply:
I have renamed